### PR TITLE
Add CarlottaSartore as maintainer

### PR DESCRIPTION
**Pull request by lrapetti**
Adding @CarlottaSartore

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @lrapetti
+* @lrapetti CarlottaSartore
```
Comment (CarlottaSartore) [inline .github/CODEOWNERS]: is this missing the @ ? i.e. @CarlottaSartore

**Review by CarlottaSartore — APPROVED** [review]
Only a single comment

File: .github/CODEOWNERS (hunk 1 above)
Comment (traversaro) [reply .github/CODEOWNERS]: Yes!

Comment (traversaro) [inline .github/CODEOWNERS]:
```suggestion
* @lrapetti @CarlottaSartore
```

**Comment by lrapetti** [comment]
Thanks for noticing it, I have fixed it!

**Pull request merged**